.changeset/clean-ears-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/build-utils': minor
```

Comment:
If it's a fix then I would expect "patch". Is there a reason "minor" was used in particular?

Reply:
Whoops, fixed to patch now. Thanks for catching that!